src/sentry/integrations/gitlab/client.py

```diff
@@ -61,7 +63,7 @@ def get_group(self, group):
         return self.get(path)
 
 
-class GitLabApiClient(IntegrationProxyClient):
+class GitLabApiClient(IntegrationProxyClient, RepositoryClient):
     integration_name = "gitlab"
```

Comment:
This can be set during the init from the integration. Just for enforcing consistency.